Example/Vouched.xcodeproj/project.pbxproj

```diff
@@ -489,7 +489,7 @@
 		};
 		D76D280F24D56C79009B6362 /* Development */ = {
 			isa = XCBuildConfiguration;
-			baseConfigurationReference = 6E60E4DFB42EE123442A8016 /* Pods-Vouched_Example.development.xcconfig */;
+			baseConfigurationReference = 847A0FB742CD0BA1EF1B22CB /* Pods-Vouched_Example.development.xcconfig */;
```

Comment:
All just config changes internal to Xcode. The important part here is just making sure that there' no public key or testing url left over